backup.tf

```diff
@@ -0,0 +1,40 @@
+resource "google_storage_bucket" "backup_repository" {
+  name          = local.backup_repository
+  location      = var.region
+  storage_class = "STANDARD"
+  force_destroy = true
```

Comment:
I would remove this line, you want to be really sure if you want to delete backups, even if you are sure that you want to remove ES cluster. Would prefer error and manual bucket deletion here. BTW: we don't utilize a special bucket for ES backup in our project (that's why `"base_path": "es_backup"`), we share a bucket with other backups. But this is just a note, no problem with creating a bucket here ;)

Reply:
But... can we make this optional somehow? Because if we set `local.backup_repository` to the existing bucket, this will throw error, so this is not compatible with our current flow

Reply:
I have added `var.backup_repository_create` variable to make this optional

Reply:
It didn't helped that much during testing, cause I did not have any backup bucket.